packages/api-clients/open-api/authorizationApi.yml

```diff
@@ -1084,32 +1093,29 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
-  "/producerKeychains/{producerKeychainId}/users/{userId}":
-    parameters:
-      - $ref: "#/components/parameters/CorrelationIdHeader"
-      - name: producerKeychainId
-        in: path
-        description: The Producer Keychain id
-        required: true
-        schema:
-          type: string
-          format: uuid
-      - name: userId
-        in: path
-        description: The identifier of the user between the security user and the consumer
-        required: true
-        schema:
-          type: string
-          format: uuid
-    delete:
-      description: Removes a user from a Producer Keychain
+    post:
       tags:
         - producerKeychain
-      summary: Remove a user from a Producer Keychain
-      operationId: removeProducerKeychainUser
+      summary: Add users to a Producer Keychain
+      operationId: addProducerKeychainUsers
+      requestBody:
+        required: true
+        description: List of users ids to add
+        content:
+          application/json:
+            schema:
+              type: array
```

Comment:
so far payloads have been objects or array of objects, because they are both easier to extend and to read.
I would suggest to use here as well an object like 
```
{
  "userIds": [""]
}
```